src/components/List/index.ts

```diff
@@ -1,3 +1,3 @@
 export { List } from './List';
-export { ListItem } from './components/ListItem';
-export { ListItemText } from './components/ListItemText';
+export { ListItem } from './components';
+export { ListItemText } from "./components";
```

Comment:
need to change " to '

Reply:
formatter generally does this as soon as I save, let me check